app/components/govuk_component/inset_text.rb

```diff
@@ -1,14 +1,18 @@
 class GovukComponent::InsetText < GovukComponent::Base
   attr_accessor :text
 
-  def initialize(text:, classes: [], html_attributes: {})
+  def initialize(text: nil, classes: [], html_attributes: {})
     super(classes: classes, html_attributes: html_attributes)
 
     @text = text
   end
 
   def call
-    tag.div(class: classes, **html_attributes) { @text }
+    tag.div(class: classes, **html_attributes) { @text || content }
```

Comment:
If both a block and `text` are specified, would it be more conventional to prioritise the block? (I think this is what the [rails `content_tag` does](https://github.com/rails/rails/blob/main/actionview/lib/action_view/helpers/tag_helper.rb#L320-L327)). Alternatively it could throw an exception?